packages/canon/src/css/stories.module.css

```diff
@@ -0,0 +1,16 @@
+.fakeBox {
```

Comment:
What's this now? 😅, could prolly use a comment to explain what's going on

Reply:
This is just a shared CSS used in multiple stories for mocked data in layout components. Not something we ship.